prql-java/prql-api/src/main/java/org/prqllang/PrqlCompileOption.java

```diff
@@ -0,0 +1,98 @@
+package org.prqllang;
+
+/**
+ * This class represents compilation options supported by prqlc. Besides the
+ * ones defined in {@code prql_compiler::Options}, more options like
+ * {@code exception} were added for Java binding.
+ */
+public class PrqlCompileOption {
```

Comment:
```suggestion
public class PrqlCompileOptions {
```

Reply:
Thanks @aljazerzen! Just renamed it to `PrqlCompileOptions`.